src/plugins/home/server/plugin.ts

```diff
@@ -36,13 +39,17 @@ export class HomeServerPlugin implements Plugin<HomeServerPluginSetup, HomeServe
     const router = core.http.createRouter();
     registerRoutes(router);
 
+    this.customIntegrations = plugins.customIntegrations;
     return {
-      tutorials: { ...this.tutorialsRegistry.setup(core) },
+      tutorials: { ...this.tutorialsRegistry.setup(core, plugins.customIntegrations) },
       sampleData: { ...this.sampleDataRegistry.setup(core, plugins.usageCollection) },
     };
   }
 
   public start(): HomeServerPluginStart {
+    if (!this.customIntegrations) {
```

Comment:
In what situation would `customIntegrations` be missing? It's a required dependency for the `home` plugin and the plugin would fail `setup` anyway if `customIntegrations` wasn't there.
I see this as an extreme edge case, but in the event of an error thrown, we probably want to be a bit more careful about the error we're throwing. The Home plugin is one of the very first experiences new users have in Kibana.

Reply:
agreed. see https://github.com/elastic/kibana/pull/112481#discussion_r715181286 for context